packages/node/src/core/evm/requests/event-logs.test.ts

```diff
@@ -1,3 +1,5 @@
+import { BLOCK_COUNT_HISTORY_LIMIT, BLOCK_COUNT_IGNORE_LIMIT } from '../../config/defaults';
```

Comment:
Maybe it would be better to make another test to test this constant then? From maintainability perspective it's better to have those "magic numbers" outside methods. Let me know what you think.

Reply:
Yeah that's the downside of hardcoding the numbers in the tests. I'm also fine with testing the constants in a `constants.test.ts` file then leaving these imports here.

The important thing to me is that if someone changes a constant value, then at least 1 test should break 👍🏻

Reply:
added test file